doc/source/installation.rst

```diff
@@ -1,140 +1,215 @@
 
-If you intend to use MDAnalysis in JupyterLab, you will have to install
-an extra package for the progress bar in analysis classes:
+If you already use `conda`_, you can add `mamba`_ to your base environment:
 
 .. code-block:: bash
 
-    conda install -c conda-forge nodejs
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    conda install -c conda-forge mamba
 
-pip
-=====
-The following command will install or upgrade the latest stable version of MDAnalysis via ``pip``, with core dependencies. This means that some packages required by specific analysis modules will not be installed.
+Then create and activate an environment with MDAnalysis:
+
+.. code-block:: bash
+
+    mamba create -n mdanalysis mdanalysis
+    mamba activate mdanalysis
+
+.. _pip-installation:
+
+pip 
+===
+
+The following command will install or upgrade the latest stable version of MDAnalysis via `pip`_ with core dependencies. 
+This means that some packages required by specific analysis modules will not be installed.
 
 .. code-block:: bash
 
     pip install --upgrade MDAnalysis
 
-If you need to install a fully-featured MDAnalysis, add the ``analysis`` tag. As with ``conda``, this will not install external programs such as `HOLE`_.
+MDAnalysis offers several optional dependency groups (“extra tags”) that you can install with pip to enable additional features:
+
+- ``analysis``: optional dependencies for various analysis modules.
+- ``extra_formats``: support for additional file formats.
+- ``parallel``: install dask to enable parallel analysis across multiple cores or machines.
+- ``doc``: install documentation build dependencies (e.g. Sphinx, themes).
+
+To see the full list of currently available extras, check the ``[project.optional-dependencies]`` dependencies section in the `pyproject.toml file`_.
+
+To install with optional features, use pip with one or more extras:
 
 .. code-block:: bash
 
-    pip install --upgrade MDAnalysis[analysis]
+    pip install --upgrade MDAnalysis[analysis,extra_formats,parallel]
 
-To install/upgrade tests:
+To install or upgrade the :ref:`test suite <mdanalysistests>` (optional, ~90 MB):
 
 .. code-block:: bash
 
     pip install --upgrade MDAnalysisTests
 
-If you intend to use MDAnalysis in JupyterLab, you will have to install
-an extra package for the progress bar in analysis classes:
+If you're using MDAnalysis in **JupyterLab**, install `ipywidgets`_ for progress bar support:
 
 .. code-block:: bash
 
-    pip install nodejs
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    pip install ipywidgets
 
+.. _source-installation:
 
-Development versions
-====================
-To install development versions of MDAnalysis, you can compile it from source. In order to install from source, you will need ``numpy`` and ``cython``. See :ref:`create-virtual-environment` for instructions on how to create a full development environment.
+Install from source
+===================
```

Comment:
Is this as a separate section really necessary? Can't we just have the developer installation (and eventually mention not to use the editable `-e` option)? I would unify the two sections, but this might just be personal preference so feel free to ignore if no-one else feels the same.

Reply:
I would keep them separate. The "Install from source" has information about the package branches. 

The "Develop installation" only uses the develop branch. 

Btw, what's so bad about `pip -e` ?

Reply:
> and eventually mention not to use the editable -e option

Sorry, I was unclear. There is nothing wrong with -e. What I was thinking was that we could have a single set of instructions and mention the different branches, and that if one is not interested in developing (i.e. editing the source), the -e option is not necessary.

But I'm OK to keep the sections separate, I just thought there was a lot of overlap.